src/types/featureFlags.ts

```diff
@@ -9,9 +9,9 @@ export interface FeatureFlags {
   }
   [FEATURE_FLAGS.NPS_FORM]: boolean
   [FEATURE_FLAGS.HOMEPAGE_ENABLED_BLOCKS]: string[]
+  [FEATURE_FLAGS.RTE_ENABLED_BLOCKS]: string[]
```

Comment:
is this type actually correct? appears to be `{blocks: string[]}`? (or we could make it stricter and give `{
    blocks: RteBlock[]
}`)

Reply:
Good catch! Hmm, I followed `HOMEPAGE_ENABLED_BLOCKS` which has the exact same structure and both flags work as expected. Will refactor it. 

Wondering how the parsing works without any issue, does it mean the type is not being enforced here?

Reply:
@seaerchin sure, added the correct type in b28f0e0daf9e2b01cde1c9295b161fb9bc2e3908

`HOMEPAGE_ENABLED_BLOCKS` need some similar refactoring but will do it in a separate PR.

Reply:
sure, can we just do `{blocks: string[]}` for `HOMEPAGE_ENABLED_BLOCKS` first? good enough first cut

Reply:
@seaerchin did some further refactoring to **actually use the defined interface type** when checking the feature flag, and added type assertion to blocks array for extra safety. See the latest change in 728925abc7580309ca1f98bc4dc9f98486a9ce0c
```
const rteEnabledBlocks = useFeatureValue(FEATURE_FLAGS.RTE_ENABLED_BLOCKS, {
    blocks: [],
  }).blocks as RTEBlockValues[]

  const isRteEnabledBlocksEmpty = rteEnabledBlocks.length === 0
```

Reply:
> sure, can we just do `{blocks: string[]}` for `HOMEPAGE_ENABLED_BLOCKS` first? good enough first cut
Of course! Did it in 20ca08c820f58b418ce1f84f1c8a07c0518a0c18